test/e2e/tests/petnames-signature-request.spec.js

```diff
@@ -0,0 +1,232 @@
+// TODO(dbrans): When petnames is fully enabled, make this file the new signature-request.spec.js.
+const { strict: assert } = require('assert');
+const {
+  withFixtures,
+  regularDelayMs,
+  openDapp,
+  DAPP_URL,
+  defaultGanacheOptions,
+  unlockWallet,
+  WINDOW_TITLES,
+} = require('../helpers');
+const FixtureBuilder = require('../fixture-builder');
+
+const signatureRequestType = {
+  signTypedData: 'Sign Typed Data',
+  signTypedDataV3: 'Sign Typed Data V3',
+  signTypedDataV4: 'Sign Typed Data V4',
+};
+
+const testData = [
+  {
+    type: signatureRequestType.signTypedData,
+    buttonId: '#signTypedData',
+    verifyId: '#signTypedDataVerify',
+    verifyResultId: '#signTypedDataVerifyResult',
+    expectedMessage: 'Hi, Alice!',
+    verifyAndAssertMessage: {
+      titleClass: '.request-signature__content__title',
```

Comment:
Were these added in this PR?

I'm not sure if I'd encourage this format as many of the selectors are only used once so we're adding a tier of abstraction when reading through a test and potentially limiting readability.

For genuine data like mock strings for inputs, I think traditional constants make perfect sense but in terms of abstracting the selectors, I think the direction we want to go in the long term is similar to mobile through the use of page model objects to abstract specific UIs. Though this is in the design and approval phase I believe.